### DISCUSSION TITLE
LN Channel with ACINQ node | why is max HTLC set to 0,001 ?

### DISCUSSION THREAD

**Discussion by Satoshi-Samourai** [ACINQ/phoenix#520] [Improving Phoenix]
Hello, 
I already wrote an email to ACINQ's node operators a couple of days ago. To no avail. I am trying my luck here, now.

We opened a 5 Mio Sats channel to the ACINQ node some days ago, just to find out that the maximum HTLC is set to 0,001 Sat on their side. This funnels the channel into complete uselessness and for a public routing node this makes no sense at all. 

Is there a mistake or are scripts in action who prefent our channel from routing? It would have been nice to know this before opening the channel, to avoid paying fees and ending up with a zombie channel. 

I am happy to get in contact with one of your node operators to discuss.

Thank you very much for filling me in on this matter.
I am looking forward to hear from you as soon as possible.

regards
Gee

**Comment by t-bast** [comment]
> We opened a 5 Mio Sats channel to the ACINQ node some days ago, just to find out that the maximum HTLC is set to 0,001 Sat on their side. This funnels the channel into complete uselessness and for a public routing node this makes no sense at all.

This is not a mistake. You opened a channel to our node: initially, all the funds are on your side, which means it's impossible to relay HTLC from the ACINQ node to your node. We let the network know that fact by setting our maximum htlc amount to 1 msat. Then, as payments flow from your node to ours, we will emit a new `channel_update` that increases this value. This ensures that senders don't try channels that will obviously not work.

The maximum htlc amount is _directional_. Our node setting a low value only applies to the direction ACINQ -> your node. You are free to set whatever value you think is best for the direction your node -> ACINQ.

See https://github.com/ACINQ/eclair/pull/2703 for more details.
